_episodes/04-drawing-bitwise.md

```diff
@@ -286,39 +249,36 @@ original image and create the mask in the same way as before:
  * Python program to apply a mask to an image.
  *
 '''
-import cv2
 import numpy as np
+import skimage
+from skimage.viewer import ImageViewer
 
 # Load the original image
-image = cv2.imread(filename = "maize-roots.tif")
+image = skimage.io.imread("maize-roots.tif")
 
-# Create the basic black image 
-mask = np.zeros(shape = image.shape, dtype = "uint8")
+# Create the basic mask 
+mask = np.ones(shape = image.shape[0:2], dtype = "bool")
```

Comment:
Nitpicking: remove whitespaces for PEP8

Reply:
i have planned to run black over all scripts once the review is over, but in order to highlight only significant changes, omitted to do so for now